MAINTAINERS.md

```diff
@@ -3,10 +3,12 @@
 ## Maintainers
 | Maintainer | GitHub ID | Affiliation |
 | --------------- | --------- | ----------- |
+| afazel | [afazel](https://github.com/afazel) | Amazon |
```

Comment:
Should we put people's real names here?

Reply:
We use people's Github username in case not everyone is willing to disclose the real name on Github publicly.